_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -112,44 +120,55 @@ This rule does not apply to `svg` elements, even inside an `html` element.
 
 #### Inapplicable Example 3
 
-This rule does not apply to `html` elements with an empty (`""`) `xml:lang` attribute.
+This rule does not apply to `math` elements.
 
-```html
-<html lang="fr" xml:lang=""></html>
+```xml
+<math xml:lang="en"></math>
 ```
 
 #### Inapplicable Example 4
 
-This rule does not apply to `html` elements whose `lang` attribute is not a [valid language subtag][], such as the empty string.
+This rule only applies to `html` in [top-level browsing context][]
 
 ```html
-<html lang="" xml:lang=""></html>
+<iframe srcdoc="<html lang='en' xml:lang='en'></html>" />
 ```
 
 #### Inapplicable Example 5
 
-This rule does not apply to `html` elements whose `lang` attribute is not a [valid language subtag][], such as only ASCII whitespace.
+This rule only applies to documents with a [content type][] of `text/html`
 
-```html
-<html lang=" " xml:lang=""></html>
+```xhtml
```

Comment:
I don't think our site knows how to handle this. @jkodu?

Reply:
It does.
https://github.com/act-rules/act-rules-web/pull/86